vgaidarji/github-android-app
# Setup GitHub API and fetch profile info

### What has been done
- Configured GitHub API and passing API token from `local.properties`
- Externalized `profile` into dedicated feature module
- Refactored build files to avoid build configuration parameters duplication between modules. See `AppConfig` in `buildSrc`  https://github.com/vgaidarji/github-android-app/pull/3/files#diff-6517152d680e852b7878bdce973c4bbde6d80ff64ccf44a7dd3570daaf95e554


### How to test
- Start the application and move to Profile tab
- Confirm user name is shown for the `vgaidarji` (hardcoded for now) GitHub user
